src/Tests/IntegrationTests/TestBasic.cs

```diff
@@ -1658,7 +1658,7 @@ public void TestNatsConn()
             }
         }
 
-        [Fact]
+        [Fact(Skip = "HANGS SOMETIMES")]
```

Comment:
@ColinSullivan1 Do you have any ideas why this one is hanging sometimes?

Reply:
I don't, it works well locally, but seems to hang up in AppVeyor.